### PR TITLE
update project templates for preview

### PR DESCRIPTION
**Pull request by aviv**
cf. https://github.com/dobtco/screendoor-v2/pull/1620

### CODE REVIEW

**Comment by aviv** [comment]
@ajb care to review/merge?

Also, it's a bit weird that the app looks so different now... I guess it's just bad timing that I just retook all the screenshots.

File: contents/articles/screendoor/projects/templates.md [hunk 1]
```diff
@@ -24,6 +24,14 @@ From the "Project templates" page, you can view and manage your project template
 
 ![project templates](../images/project_templates.png)
 
+## Previewing templates
+
+To preview a template, just click on it's name from the "Project templates" page.
```
Comment (ajb) [inline contents/articles/screendoor/projects/templates.md]: :scissors: apostrophe

**Comment by ajb** [comment]
Looks good besides the one comment. As for the UI looking different, keep in mind we probably notice 100x more changes than the average user.

**Comment by aviv** [comment]
Forsure. Can't believe that apostrophe didn't jump out at me... must be losing it.

**Pull request merged**